docs/pages/color_examples.js

```diff
@@ -0,0 +1,388 @@
+              </Box>
+            </Flex>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use the established [typography](/color_usage#Typography-color) and [iconography](/color_usage#Iconography-color) color tokens so users can quickly scan and identify sentiment."
+            defaultCode={`
+            <Status
+              type="ok"
+              title="Campaign complete"
+            />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Apply alternative colors to text and icons. Always refer to [color usage](/color_usage) for the appropriate color pattern. "
+            defaultCode={`
+            <Flex gap={1} alignItems="center">
+              <Icon icon="eye"/>
+              <Text weight="bold">
+                <Box dangerouslySetInlineStyle={{
+                  __style: {color: 'mediumpurple'}
+                }}>
+                  views
+                </Box>
+              </Text>
+            </Flex>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use the appropriate [color tokens](/design_tokens) to switch between themes (light and dark mode). It ensures consistency and avoids accessibility issues."
+            defaultCode={`
+            <Flex direction="column" gap={8} alignItems="center">
+              <Flex gap={4}>
+                <IconButton icon="speech" />
+
+                <IconButton icon="share" iconColor="darkGray" />
+              </Flex>
+              <Flex gap={4}>
+                <Button color="red" text="Primary" />
+                <Button color="gray" text="Secondary" />
+                <Button color="blue" text="Shop" />
+              </Flex>
+              <Flex gap={4}>
+                <SearchField
+                  accessibilityLabel={'Search you Pins'}
+                  id={'color-do-search'}
+                  placeholder="Search your Pins"
+                  onChange={() => {}}
+                />
+              </Flex>
+              <Flex gap={8}>
+                <Text>Default text</Text>
+                <Text color="subtle">Subtle text</Text>
+              </Flex>
+            </Flex>
+`}
+          />
+
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Apply alternative colors not specified in our color tokens when switching between themes. If a new color value is needed for a specific use case, [let the team know](https://gestalt.netlify.app/how_to_work_with_us#Meetings-and-events) and we will evaluate."
+            defaultCode={`
+            <ColorSchemeProvider colorScheme="dark" id="dark-example-dont">
```

Comment:
I'd use this example in the Do, and the Don't use "alternative" colors to show the problem

Reply:
This one is using alternative colors, but I can make it more obvious